site/public/css/server.css

```diff
@@ -275,6 +276,21 @@ table tr.table-header {
     vertical-align: middle;
 }
 
+.table-left {
```

Comment:
Name should be more clear. `table-left` does not signifies that all fonts are to left

Reply:
According to me, previous configuration was better as first name was easily differentiable between section and first name. Apart from this if we can also change header position still it will create confusion when zoomed in (or for small screen)

Reply:
I agree that the first name was easily differentiable between section and first name. But the increase in readability is more important as several people will glance at the names and find the one they are looking for quickly in a left aligned layout.

As for the name, I think it can be improved to `table-text-left` so It specifies exactly what it is intended to do. I will add this in my next commit.